controls/togglebutton/ripple.md

```diff
@@ -13,23 +13,23 @@ Apply a ripple effect to the ToggleButton, by using the following attached prope
 
 * `RippleColor` (`Color`)&mdash;Specifies the color of the ripple effect.
 * `RippleMode` (enum of type `Telerik.Maui.Theming.RippleMode`)&mdash;Specifies the mode in which the ripple effect can be visualized. The options are:
-	* `None`&mdash;The element will not render ripple effects.
-	* `Pulse`&mdash;The element will play a ripple effect on pressed and the ripple will fade away even on hold.
+	* (Default)`Pulse`&mdash;The element will play a ripple effect on pressed and the ripple will fade away even on hold.
```

Comment:
See the suggestions for the ripple modes in `controls/templatedbutton/ripple.md`.